ocs_ci/ocs/constants.py

```diff
@@ -708,6 +709,8 @@
 CEPHFS_CTRLPLUGIN_LABEL = "app=openshift-storage.cephfs.csi.ceph.com-ctrlplugin"
 RBD_CTRLPLUGIN_LABEL = "app=openshift-storage.rbd.csi.ceph.com-ctrlplugin"
 PROMETHEUS_POD_LABEL = "app.kubernetes.io/name=prometheus"
+CEPHFS_CTRLPLUGIN_NAME = "openshift-storage.cephfs.csi.ceph.com-ctrlplugin"
```

Comment:
Please use existing label variables in constants.py
CEPHFS_CTRLPLUGIN_LABEL
RBD_CTRLPLUGIN_LABEL